lib/commands/view.js

```diff
@@ -236,6 +236,15 @@ class View extends BaseCommand {
       }
     })
 
+    //No data has been pushed because no data is matching the specified version
+    if (data.length == 0) {
+      const er = new Error(`No match found for version ${version}`)
+      er.statusCode = 402
```

Comment:
402 is "payment required"; did you mean 404?
```suggestion
      er.statusCode = 404
```

Reply:
Yes, thank you!